packages/a11y/src/AriaLabelProps.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * A union interface requiring _either_ `aria-label` or `aria-labelledby`
+ */
+export type AriaLabelProps =
```

Comment:
Would this only be for a component that doesn't accept the `label` prop?

Reply:
Yeah, in that case you'd extend this interface

Reply:
maybe we should also expose an interface that uses `Pick` for when the three are optional

Reply:
Added `AriaLabelPropsWithLabel`

Reply:
oops, meant to say `Either` but this also works!

Reply:
I think Either only works with 2 keys. It's easier to export this from here anyway I think